packages/xchain-cosmos/src/cosmos/sdk-client.ts

```diff
@@ -43,37 +45,37 @@ export class CosmosSDKClient {
   }
 
   setPrefix(): void {
-    AccAddress.setBech32Prefix(
-      this.prefix,
-      this.prefix + 'pub',
-      this.prefix + 'valoper',
-      this.prefix + 'valoperpub',
-      this.prefix + 'valcons',
-      this.prefix + 'valconspub',
-    )
+    setBech32Prefix({
```

Comment:
This `set` could not affect other packages like `xchain-thorchain` as well if they are in the same project?. That is because you are changing/replacing global parameters